core/storage/base_model/gae_models.py

```diff
@@ -283,20 +339,21 @@ def get_all_commits(cls, page_size, urlsafe_start_cursor):
             cls.query(), page_size, urlsafe_start_cursor)
 
     @classmethod
-    def get_commit(cls, instance_id, version):
+    def get_commit(cls, target_entity_id, version):
         """Returns the commit corresponding to an instance id and
         version number.
 
         Args:
-            instance_id: str. The id of the particular construct, from which
-                this method is called.
+            target_entity_id: str. The id of the particular construct, from
```

Comment:
I would change the docstring to be more descriptive of what the "target entity ID" represents, similar to line 303 above.

Reply:
Done.